test/metabase/analytics/snowplow_test.clj

```diff
@@ -80,47 +80,46 @@
 (deftest track-event-test
   (with-fake-snowplow-collector
     (testing "Data sent into [[snowplow/track-event!]] for each event type is propagated to the Snowplow collector,
-           with keys converted into snake-case strings, and the subject's user ID being converted to a string."
-      (mt/with-temporary-setting-values [anon-tracking-enabled true]
```

Comment:
The main change in this test is just removing this line, since I've moved the setting into the `with-fake-snowplow-collector` macro. But I also added `source` fields to the `invite-sent` test cases, for completeness.